tests/subsys/emul/prj.conf

```diff
@@ -0,0 +1,10 @@
+# Copyright 2022 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+CONFIG_I2C=y
+CONFIG_SENSOR=y
+CONFIG_BMI160=y
```

Comment:
this should all be reduced to `CONFIG_SENSOR=y`, see 

https://github.com/zephyrproject-rtos/zephyr/blob/352fde1271c568fa24cb48b4aed20f97b25945b1/drivers/sensor/bmi160/Kconfig#L6-L11

Reply:
Done.